example/src/demos/Portals.tsx

```diff
@@ -110,16 +110,22 @@ const App = () => (
       {/* First layer, a portal */}
       <Portal scale={[4, 5, 1]} position={[0, 2.5, 0]} rotation={[0, 0, 0]}>
         <Lights />
-        <Farm scale={10} rotation={[0, 0, 0]} position={[-1, -2, -10]} />
-        <Soda scale={5} position={[2, -2, -1.5]} />
+        <Suspense fallback={null}>
+          <Farm scale={10} rotation={[0, 0, 0]} position={[-1, -2, -10]} />
+          <Soda scale={5} position={[2, -2, -1.5]} />
+        </Suspense>
         <Portal scale={[4, 5, 1]} position={[2, 0, -5]} rotation={[0, 0, 0]}>
           <Lights />
-          <Soda scale={8} position={[0, -2, -1.5]} />
-          <Environment preset="city" background="only" />
+          <Suspense fallback={null}>
+            <Soda scale={8} position={[0, -2, -1.5]} />
+            <Environment preset="city" background="only" />
+          </Suspense>
         </Portal>
       </Portal>
-      <Ramen scale={4} position={[-2, 0, 2]} />
-      <Soda scale={5} position={[1.5, 0, 3]} />
+      <Suspense fallback={null}>
+        <Ramen scale={4} position={[-2, 0, 2]} />
+        <Soda scale={5} position={[1.5, 0, 3]} />
+      </Suspense>
```

Comment:
This PR causes ULE in `Portal` to fire without a ref attached outside of `createPortal`.

Reply:
Looks like elements (including containers like `createPortal`) won't have their refs created/updated when the tree suspends.

```tsx
function Test() {
  suspend(async () => null, [])
  const group = React.useRef<THREE.Group>(null!)
  React.useLayoutEffect(() => console.log('Test', group.current), [])
  return <group ref={group} />
}

function Portal({ children }: { children?: React.ReactNode }) {
  const group = React.useRef<THREE.Group>(null!)
  const [scene] = React.useState(() => new THREE.Scene())
  React.useLayoutEffect(() => void console.log('portal', group.current, scene.children), [scene])
  return <group ref={group}>{children && createPortal(children, scene)}</group>
}

//
<Portal />
<Test />
```

Reply:
Everything works as expected if `commitMount` is fired in `finalizeInitialChildren` though. I've avoided doing this before, will have to read up on why exactly.